src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -322,6 +324,7 @@ private void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, KeyboardAcc
 					break;
 
 				case (false, true, true, VirtualKey.Add): // alt + shift + "+" open pane
+				case (false, true, true, PlusKey):
```

Comment:
```suggestion
				case (false, true, true, (VirtualKey)187;):
```

Is a constant needed if it's only used once?

Reply:
It is used as well in the `PaneHolderPage.xaml`